catalog/app/utils/ErrorBoundary.js

```diff
@@ -17,23 +16,22 @@ export const createBoundary = (handle, name = 'ErrorBoundary') =>
     constructor(props) {
       super(props)
       this.state = { handled: null }
+      this.boundReset = this.reset.bind(this)
     }
 
     componentDidUpdate({ id: oldId }) {
       if (this.props.id !== oldId) this.reset()
     }
 
     componentDidCatch(error, info) {
-      const { reset } = this
-      const handled = handle(this.props, { reset })(error, info)
+      const handled = handle(this.props, { reset: this.boundReset })(error, info)
       if (handled && handled !== error) {
         this.setState({ handled })
       } else {
         throw error
       }
     }
 
-    @boundMethod
```

Comment:
This doesn't work. Maybe, because of typescript. I didn't dig into it because it is used very rarely, so it's better to remove this dependency